perun-core/src/main/java/cz/metacentrum/perun/core/impl/AuditMessagesManagerImpl.java

```diff
@@ -214,12 +219,15 @@ public List<AuditMessage> getMessagesByIdAndCount(PerunSession perunSession, int
 
 	@Override
 	public Paginated<AuditMessage> getMessagesPage(PerunSession perunSession, MessagesPageQuery query) {
+		MapSqlParameterSource parameters = new MapSqlParameterSource();
+		parameters.addValue("selectedEvents", query.getSelectedEvents());
+
 		// take exact total count up to PAGE_COUNT_PRECISION entries, estimate it otherwise
-		return jdbc.query("select " + auditMessageMappingSelectQuery
+		return namedParameterJdbcTemplate.query("select " + auditMessageMappingSelectQuery
 			+ ", case when (select count(*) from (select 1 from auditer_log limit" + PAGE_COUNT_PRECISION + ") as sample) < " + PAGE_COUNT_PRECISION + " then (select count(*) from auditer_log) else " +
 			"(select 100 * count(*) FROM auditer_log TABLESAMPLE SYSTEM (1)) end as total_count "
```

Comment:
the total count should be updated as well, it should represent how many results there are after applying all the filters, not before